tests/test_issues/test_linkml_issue_1094.py

```diff
@@ -62,3 +62,8 @@ def test_pydanticgen_inline_dict():
 
     assert dict_field.default_factory is None
     assert list_field.default_factory is None
+
+    output = gen.serialize()
+    output_subset = [line for line in output.splitlines() if "has_bikes: " in line]
+    assert len(output_subset) == 1
+    assert "has_bikes: Dict[str, Union[str, Bike]] = Field(...," in output_subset[0]
```

Comment:
```suggestion
assert str(dict_field.default) == "PydanticUndefined"
assert str(list_field.default) == "PydanticUndefined"
```

how about this? the module is already serialized and compiled above, and this along with the default_factory assertions are a direct test of what you're after (ie. that it's an ellipse) without needing to be updated every time the pydantic generator changes (as u experienced) <3

Reply:
Done, thanks!